File: mobi_client/mobi_client/mobi_hotfix/mobi_hotfix/HFScripts/HFConfig/ConfigModel/ModelSkillConfigModel.cs
public class ModelSkillConfigModel : ConfigSystem.ConfigModel {
    public System.Collections.Generic.List<int> skills;//技能
    public string model_qversion;//战斗prefab
    public string init_scale;//初始缩放比例
    public System.Collections.Generic.List<string> dead_audio;//死亡音效
    public System.Collections.Generic.List<string> onhit_audio1;//普通受击音效
    public System.Collections.Generic.List<string> onhit_audio2;//受重击音效
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/ModelSkillConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class ModelSkillConfigModel extends ConfigModel {
	public ArrInt skills;//技能
	public String model_qversion;//战斗prefab
	public String init_scale;//初始缩放比例
	public ArrString dead_audio;//死亡音效
	public ArrString onhit_audio1;//普通受击音效
	public ArrString onhit_audio2;//受重击音效
}